saleor/graphql/account/mutations/account/send_confirmation_email.py

```diff
@@ -0,0 +1,134 @@
+from typing import cast
+from urllib.parse import urlencode
+
+import graphene
+from django.conf import settings
+from django.contrib.auth.tokens import default_token_generator
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+
+from .....account import models
+from .....account.error_codes import SendConfirmationEmailErrorCode
+from .....account.notifications import send_account_confirmation
+from .....core.utils.url import prepare_url, validate_storefront_url
+from .....permission.auth_filters import AuthorizationFilters
+from .....webhook.event_types import WebhookEventAsyncType
+from ....channel.utils import clean_channel
+from ....core import ResolveInfo
+from ....core.descriptions import ADDED_IN_315
+from ....core.doc_category import DOC_CATEGORY_USERS
+from ....core.mutations import BaseMutation
+from ....core.types import SendConfirmationEmailError
+from ....core.utils import WebhookEventInfo
+from ....plugins.dataloaders import get_plugin_manager_promise
+from ....site.dataloaders import get_site_promise
+
+
+class SendConfirmationEmail(BaseMutation):
+    class Arguments:
+        redirect_url = graphene.String(
+            required=True,
+            description=(
+                "Base of frontend URL that will be needed to create confirmation "
+                "URL."
+            ),
+        )
+        channel = graphene.String(
```

Comment:
IMHO, I might be wrong, but I think that the new queries/mutations always require providing the channel. Solution for single channel was something that we needed between 2.x->3.0.
I would mark it as a required.